src/device-registry/utils/create-event.js

```diff
@@ -522,28 +525,41 @@ const createEvent = {
       return await axios
         .post(constants.BULK_ADD_VALUES_JSON(channel), requestObject)
         .then(function(response) {
-          let output = JSON.parse(response.config.data).updates;
-          return {
-            message: "successfully transmitted the data",
-            success: true,
-            data: output,
-          };
+          if (isEmpty(response)) {
+            return {
+              success: false,
+              message: "successful operation but no data sent",
+              status: HTTPStatus.CONFLICT,
+              errors: {
+                message: "likely duplicate values or system conflicts",
+              },
+            };
+          } else if (!isEmpty(response)) {
+            let output = JSON.parse(response.config.data).updates;
+            return {
+              message: "successfully transmitted the data",
+              success: true,
+              data: output,
+              status: HTTPStatus.OK,
+            };
+          }
         })
         .catch(function(error) {
-          const errorMessage = error.response
-            ? error.response.data
-            : "No active internet connection";
-
           return {
             success: false,
-            message: "Internal Server Error",
+            message: "Server Error",
             errors: {
-              message: errorMessage,
+              message: error.response
+                ? error.response.data.error.details
+                : "No active internet connection",
```

Comment:
`No active internet connection` might be missing leading to the API user.  If a user is able to get a response from the API it means there was in internet connection that enabled deliver of the response which contradicts with the API response message

Reply:
Done. @NoahNsimbe